src/lib/y2security/security_policies/manager.rb

```diff
@@ -140,6 +146,35 @@ def env_policies
 
         ENV[key].split(",")
       end
+
+      SERVICE_NAME = "ssg-apply".freeze
+      private_constant :SERVICE_NAME
+
+      # Adds the package and enables the service to remedy the system after the installation
+      def enable_service
+        return if enabled_services.include?(SERVICE_NAME)
+
+        enabled_services << SERVICE_NAME
+        Yast::PackagesProposal.AddResolvables("security", :package, [SERVICE_NAME])
+      end
+
+      # Disables the service and removes the package to remedy the system after the installation
+      def disable_service
+        return unless enabled_services.include?(SERVICE_NAME)
+
+        enabled_services.delete(SERVICE_NAME)
+        Yast::PackagesProposal.RemoveResolvables("security", :package, [SERVICE_NAME])
+      end
+
+      # Return the list of enabled services
+      def enabled_services
+        require "installation/services" unless defined?(::Installation::Services)
```

Comment:
I would add a comment about cyclic dependency problem, similar to https://github.com/yast/yast-security/pull/128/files#diff-b6b853b94bce3cd38105d93a59732aa0345ed6ad7e53df5afad8e337d13bdb7cR81.

Reply:
BTW, yast2-installation only has a BuildRequires yast2-security. Could yast2-security require yast2-installation?

Reply:
And I would say that `SecuritySettings` and `Services` should not be in yast2-installation.

Reply:
I agree. I guess they were placed there for some reason. Perhaps we might consider changing them for SP5, but for SP4 I would keep them as they are.